sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -1069,6 +1069,15 @@ class SQLQuerySuite extends QueryTest with SharedSparkSession with AdaptiveSpark
     spark.sessionState.conf.clear()
   }
 
+  test("SPARK-35044: `SET propertyKey` shall lookup sharedState.hadoopConf to display the" +
```

Comment:
nit: `SPARK-35044: SET command shall display default value for hadoop conf correctly`